src/awx_plugins/interfaces/_temporary_private_inject_api.py

```diff
@@ -159,6 +159,7 @@ def inject_credential(
     safe_env: EnvVarsType,
     args: ArgsType,
     private_data_dir: str,
+    **kwargs: dict[str, str],
```

Comment:
Let's avoid kwargs when possible:
```suggestion
    *,
    container_root: str = '',
```

Reply:
I think you forgot to drop the `*,`

Reply:
I didn't. I kept what seemed to be your intent.